content/tutorials/drift-detection/index.md

```diff
@@ -0,0 +1,198 @@
++   ├─ aws:ec2:Subnet                 subnet                 created (10s)
++   ├─ aws:ec2:InternetGateway        gateway                created (0.55s)
++   ├─ aws:ec2:SecurityGroup          secGroup               created (2s)
++   ├─ aws:ec2:RouteTable             routeTable             created (0.91s)
++   ├─ aws:ec2:RouteTableAssociation  routeTableAssociation  created (0.51s)
++   └─ aws:ec2:Instance               server                 created (12s)
+
+Outputs:
+   hostname: "ec2-18-236-205-169.us-west-2.compute.amazonaws.com"
+   ip      : "18.236.205.169"
+   url     : "http://ec2-18-236-205-169.us-west-2.compute.amazonaws.com"
+
+Resources:
+   + 8 created
+```
+
+Now you can view your new stack in Pulumi Cloud by following the **View in Browser** link. Click on the **Drift** tab and you will see that Drift is unavailable as your stack's deployments are not yet configured. Click on the **Configure deployments setting** to see your Deployments source control settings.
+
+To use Drift Detection with Deployments you will first need to create a new GitHub repository.
+
+## Create a new repository on GitHub
+
+1. Go to [GitHub](https://github.com/new) and create a new repository manually.
+2. Copy the repository URL provided by GitHub.
+3. Replace the origin with your new GitHub repo using the following commands:
+
+```bash
+git init
+git add .
+git commit -m "Initial commit for my Pulumi Drift tutorial"
+git branch -M main
+git remote add origin https://github.com/your-username/learn-pulumi-drift-detection.git
+git push -u origin main
+```
+
+## Configure Drift Detection
+
+Now you will return to your browser to set your deployment configuration source control settings. Navigate to your stack `Settings` and `Deploy` tab. Next, configure the required Deployment settings, including your source control.
+
+1. Add your GitHub organization/repository and branch you created.
+
+2. Add your Secret environment variables, including `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`.
+
+{{% notes type="info" %}}
+Secret environment variables, such as `AWS_SECRET_ACCESS_KEY`, are encrypted end-to-end with Pulumi and can be set on each stack. However, by creating an [environment](/docs/pulumi-cloud/esc) with Pulumi ESC, you can centralize secrets and set up OIDC for secure authentication. This allows you to manage and share sensitive configuration data across multiple stacks efficiently.
+{{% /notes %}}
+
+Click **Save deployment configuration**.
+
+![Animation of how to configure drift detection in Pulumi](./pulumi-configure-drift.mp4)
+
+Once you have configured Pulumi Deployments, open the Actions menu and choose `Detect Drift`, then click `Deploy` to validate your stack, program and deployments are all running before we introduce drift.
+
+![Screenshot of the Pulumi Cloud console to run a Drift Detection](./pulumi-detect-drift-action.png)
+
+Once the drift run is complete, you should see a new Drift check on the stack `Drift` Tab
```

Comment:
What does mean or indicate? You should explain that given there were no deviations  because no drift had been introduced yet.

Reply:
thx; added more detail for clarity.